src/settings.html

```diff
@@ -4,39 +4,39 @@
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Settings page</title>
-		<h1>Settings</h1>
 		<!--<link rel="stylesheet" href="css/styles.css" /> -->
 	</head>
 
 	<body>
-		<div class="settingsMenuOptions">
-			<form id="settingsForm">
+        <h1 data-translate="Settings">Settings</h1>
+        <div class="settingsMenuOptions">
 				<div data-translate="Language" id="langSelDiv">
 					Language
+                </div>
 					<select class="settingsDropdowns" id="languageSelector">
 						<option value="english">English</option>
```

Comment:
1. value ar trebuie sa fie "en" si "ro" in loc de "english" si "romanian"

2. Lipsesc traducerile la options. Pune data-translate pentru option sa traducem romana/engleza.